### DISCUSSION TITLE
Darktrace and SentinelOne integrations not populating data

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#10821] [2.4]
2.4 Beta4 
Rocky 9
Standalone
I have installed the elastic agent integrations for Darktrace and Sentinel one and although I have it all installed and the EFleet server is looking healthy, I am unable to see any data received from new integrations.
I am not showing any errors. Any thoughts?

**Comment by dougburks** [comment]
Have you checked the log files in /opt/so/log for additional clues?

**Reply by Phillip-Samson** [reply]
The issue was that I was not. putting https:// in front of the URLs 
Now that I put in the https:// SentinelOne is working.
I found the error in the agent logs by running the download diag in Fleet Server Agent info
Darktrace is still failing but now it's due to an untrusted x509 

Now I am struggling to get the CA cert for our PKI working with the Darktace  integration 
I tried adding the root cert, the root and the subordinate, and the root subordinate and server cert in the DT integration config but nothing seems to work at the moment.
any thoughts on this?

**Reply by Phillip-Samson** [reply]
I was able to get the cert working. You must maintain the exact same indentations and overall formatting as what is presented in the integration module example. Remember it is CRLF not LF when copying and pasting I used VSCode to manage this edit.
 

Now what I am seeing is that the pipelines work when I test them with a document that is already indexed, but the data is not being parsed properly during the ingest phase.
